python/kserve/test/test_server.py

```diff
@@ -73,7 +77,12 @@ def load(self):
         self.ready = True
 
     async def predict(self, request, headers=None):
-        return {"predictions": request["instances"]}
+        if isinstance(request, InferRequest):
+            inputs = get_predict_input(request)
+            infer_response = get_predict_response(request, inputs, self.name)
+            return infer_response.to_rest()
```

Comment:
`to_rest` should be called in `postprocess` handler automatically if the request includes the `application/json` content type header, so I think we just need to return `InferResponse` here.
https://github.com/kserve/kserve/blob/eb794b5ab35f63564a04df02546c653b4cafd3aa/python/kserve/kserve/model.py#L250

Reply:
updated!